.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,40 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+<!--
+Thank you for taking the time to file an issue, we highly appreciate your contribution.
+
+Before you go ahead, please make sure that the issue has not been reported already.
+For support questions, please head over to our [Discord](http://near.chat/) or ask on [StackOverflow](https://stackoverflow.com/search?q=nearprotocol).
+-->
+
+**Describe the bug**
+Please provide a short description of the bug.
+
+**To Reproduce**
```

Comment:
It feels like this part is for something frontend related and doesn't really apply here

Reply:
Some people might feel more comfortable to describe the bug? Could you change the template to better fit your needs?